packages/dds/cell/src/cell.ts

```diff
@@ -227,7 +227,8 @@ export class SharedCell<T = any>
 	protected async loadCore(storage: IChannelStorageService): Promise<void> {
 		const content = await readAndParse<ICellValue>(storage, snapshotFileName);
```

Comment:
Same comment as directory - probably better to just use `readBlob` + `this.serializer.parse`